packages/edit-post/src/components/preferences-modal/enable-custom-fields.js

```diff
@@ -39,8 +39,7 @@ export function CustomFieldsConfirmation( { willEnable } ) {
 				) }
 			</p>
 			<Button
-				// TODO: Switch to `true` (40px size) if possible
-				__next40pxDefaultSize={ false }
+				__next40pxDefaultSize
```

Comment:
- Edit a post
- Open the preferences modal
- Toggle the "Custom fields" option in the "Advanced" section

![Screenshot 2024-10-01 at 22 03 00](https://github.com/user-attachments/assets/8cb8ed2c-844e-4429-be17-8a4117947a42)